sources/T-Editing-Guidelines-201602.adoc

```diff
@@ -0,0 +1,843 @@
+= Author's guide for drafting ITU-T Recommendations
+ITU
+:bureau: T
+:docnumber: EDITING GUIDELINES
+:series:
+:series1:
+:series2: 
```

Comment:
Do we need these three lines of `:seriesX:`? If not we can remove them.

Reply:
Removing

Reply:
Yes, remove them from the Editing Guidelines, but only because the Editing Guidelines don't have associated series.